adagrams/game.py

```diff
@@ -1,11 +1,135 @@
+import random
+
 def draw_letters():
-    pass
+ 
+    NUM_TILES = 10
+    letters_grabbed = []
+    letters_remaining_pool = LIST_OF_LETTERS.copy()
+    
+    for i in range(NUM_TILES):
+        # update the last index of the remaining pool list that changes everytime
+        # random.randint stop is inclusive, so -1 which reflect the size of the pool
+        index = len(letters_remaining_pool)-1
```

Comment:
Missing white space around the subtraction operator, should be `len(letters_remaining_pool) - 1`

It's a small thing, but as you write more code (and eventually contribute to a large codebase) it's important to be consistent with whitespaces so that the whole project stays neat. Here's the official Python [style guide](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) that covers whitespaces.